scripts/dataflow_copy.py

```diff
@@ -36,15 +95,17 @@
         for session in sessions:
             # set source dir - NAS
             trg_dir = os.path.join(src_dir, session)
-            # set dest dir - neo if subject id odd, else drwho
-            subj_id = int(session.split('_')[0])
-            if subj_id % 2: #odd
-                dest_dir = os.path.join(dest_dir_1, session)
-            else: #even
-                dest_dir = os.path.join(dest_dir_2, session)
-
-            # Note: trg_dir and dest_dir do not have trailing slashes here!
-            copy_files(trg_dir, dest_dir, db_table, sensor_file_table)
+            # check if session is already copied
```

Comment:
comment may not be needed as it's the same as the function name.

Reply:
Changed+added comments to indicate code logic